### DISCUSSION TITLE
Can´t update

### DISCUSSION THREAD

**Discussion by LoveBorjesson** [iterate-ch/cyberduck#16512] [Q&A]
<img width="266" alt="Skärmavbild 2024-11-08 kl  15 55 42" src="https://github.com/user-attachments/assets/68319918-a32d-49e8-8452-107e74e93049">

Hello!

I can not update because CyberDuck wants to connect to (ftp.servage.net – FTP). I don't have that server anymore.
I have two ftp-servers. ftp.wasabryggeriet.se and ftp.love-music.se

Can You erase ftp.servage.net?

I have a paid version of CyberDuck.

Sincerely
Love Börjesson
Sweden

**Comment by dkocher** [comment]
As the message indicates move _Cyberduck.app_ from your _Downloads_ folder to _/Applications_.
